scripts/bert/index.rst

```diff
@@ -319,153 +319,14 @@ Command line interface
-+-----------------------+----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+
-| epochs                | 2                                                                                                                          | 2                                                                                                                           | 2                                                                                                                           |
-+-----------------------+----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+
-| training log          | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluonnlp/logs/bert/static_finetune_squad1.1_base.log>`__      | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluonnlp/logs/bert/static_finetune_squad1.1_large.log>`__      | `log <https://raw.githubusercontent.com/dmlc/web-data/master/gluonnlp/logs/bert/static_finetune_squad2.0_large.log>`__      |
-+-----------------------+----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+
-| command               | [8]                                                                                                                        | [9]                                                                                                                         | [10]                                                                                                                        |
-+-----------------------+----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+
+    $ python export/export.py --task classification --model_parameters /path/to/saved/ckpt.params --output_dir /path/to/output/dir/ --seq_length 256
```

Comment:
why 256 here? i find 128 is a default value in finetune_classification.py.

Reply:
I just picked an arbitrary number. I can update it